specification/datadog/resource-manager/Microsoft.Datadog/stable/2022-08-01/datadog.json

```diff
@@ -0,0 +1,2013 @@
+{
+  "swagger": "2.0",
```

Comment:
Please provide a link to the PR from the private repo if this was already reviewed. Also submit any delta between the already reviewed PR and the current one as a separate commit so only those parts may be reviewed

Reply:
Private repo revied PR: https://github.com/Azure/azure-rest-api-specs-pr/pull/10646
Final code changes in private repo: https://github.com/Azure/azure-rest-api-specs-pr/blob/main/specification/datadog/resource-manager/Microsoft.Datadog/stable/2021-03-01/datadog.json

I have added a base commit with old changes and then pushed the new API version changes.